api-operator/pkg/mgw/hpa.go

```diff
@@ -140,7 +141,7 @@ func ValidateHpaConfigs(client *client.Client) error {
 	if hpaConfMap.Data[hpaVersionConst] == "v2beta1" {
```

Comment:
Shall make this as
```suggestion
	if strings.EqualFold("v2beta1", hpaConfMap.Data[hpaVersionConst])  {
```

and next if condition as  `else if` and finally another `else` with logging an error with saying "invalid HPA version" and return an error with `errors.New("")`.

Reply:
Fixed with 12b10f3c97ac27e7ec8d0944e114a660ce1a8f7a